docs/migration/manual-migration.mdx

```diff
@@ -38,6 +40,24 @@ grep "module.exports" src/ -r
 
 :::
 
+#### Updating the project's `package.json`
+
+This transition will also require your project to be declared as a `module`. In
+order to do so, you can add the following to your `package.json`:
+
+```json
+  // ...
+  "name": "my-project",
+  // highlight-next-line
+  "type": "module",
+  //...
+```
+
+#### Uses of `__dirname`
+
+`__dirname` isn't available anymore in ESM. To cope with this change, you will
```

Comment:
Given how variable a use of `__dirname` can be, I wasn't comfortable about giving replacement suggestions here.

Reply:
What do you mean by "variable" in esm it just replaced with meta

```ts
import { dirname } from 'path';
import { fileURLToPath } from 'url';

const __dirname = dirname(fileURLToPath(import.meta.url));
```

Reply:
**nitpick:** I don't think it's relevant to document a lot of things here. An example is ok, but not more.

**suggestion:** for details I think it's better to link to one (or several) of articles published on the web … this change has been around for 10 years. There are lots of resources. Example: https://blog.logrocket.com/alternatives-dirname-node-js-es-modules/

Reply:
Added Paul's example and updated the section with links.